index.js

```diff
@@ -131,6 +138,9 @@ class GooglePlaceAutocomplete extends Component {
 
                     // Fire event
                     if (this.props.onResult) {
+                        this.setState({
+                          predictions:[],
+                        });
```

Comment:
Could you move this outside the check for `this.props.onResult`?
You could also use `this._predictions([])` here instead so that `this.props.onPredictions` can be triggered.